src/Components/Aspire_Components_Progress.md

```diff
@@ -10,6 +10,7 @@ As part of the .NET Aspire November preview, we want to include a set of .NET As
 | Microsoft.Data.SqlClient                |                  ✅                  |             ✅             |                           ✅                            |              ✅              |          ❌          |        ✅            |         ✅           |              ✅                  |
 | Microsoft.EntityFramework.Cosmos        |                  ✅                  |             ✅             |                           ✅                            |              ✅              |          ✅          |        ✅            |         ❌          |              ❌                  |
 | Microsoft.EntityFrameworkCore.SqlServer |                  ✅                  |             ✅             |                           ✅                            |              ✅              |          ✅          |        ✅            |         ✅           |              ✅                  |
+| MongoDB.Driver |                  ✅                  |             ✅             |                           ✅                            |              ✅              |          ❌          |        ❌            |         ❌           |              ✅                  |
```

Comment:
Can you format these lines so the table is readable in text view?

```suggestion
| MongoDB.Driver                          |                  ✅                  |             ✅             |                           ✅                            |              ✅              |          ❌          |        ❌            |         ❌           |              ✅                  |
```

Reply:
This has logging right? Why is that column an X?

Reply:
I've added the logging support after this file, I'll update that